### ISSUE TITLE
aardvark collection info tab shows incorrect alive count

### ISSUE THREAD

**Issue by dsonet** [arangodb/arangodb#1921]
Version: 3.0.1
System: Windows 10
Somehow it shows different as what I can see from content tab. Even count the alive and dead still doesn't match.

**Comment by dothebart** [comment]
Can you provide a little more details? The actual number of documents in your collection (`db.collectionname.count()` in arangosh), and a screenshot of the aardvark with the wrong figures?

**Comment by jsteemann** [comment]
@dsonet: please note that the figures output only reflects documents & data that are in the collection's datafiles. Any data from the collection that is still located in the WAL (write-ahead log) is not shown there. Data is moved from the WAL to the collection datafiles eventually, but not directly after each data modification operation. So there will always be differences between the figures output and the number of documents in the collection. Additionally, data in datafiles may be compacted later, in which case old and now unneeded document revisions may get removed.
Long story short: figures().alive is not supposed to be equal to collection.count() all the time.

**Comment by Simran-B** [comment]
Here's some code you can try in arangosh:

```js
var internal = require("internal")
db._drop("aliveCount")
var coll = db._create("aliveCount")

db._query(aql`FOR i IN 1..100000 INSERT {i} INTO ${coll}`)
print(`Count: ${coll.count()}, Alive: ${coll.figures().alive.count} after INSERT`)

internal.wal.flush()
print(`Count: ${coll.count()}, Alive: ${coll.figures().alive.count} after flush()`)

internal.wait(1)
print(`Count: ${coll.count()}, Alive: ${coll.figures().alive.count} after waiting a bit`)
```

Example output:

> Count: 100000, Alive: 0 after INSERT
> Count: 100000, Alive: 0 after flush()
> Count: 100000, Alive: 100000 after flush() and waiting a bit

**Comment by dothebart** [comment]
Closing this question as answered. If there are still bits unexplained, don't hesitate to ask.
